src/IO/H5/StellarCollapseEos.hpp

```diff
@@ -0,0 +1,82 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+/// \file
+/// Defines class h5::StellarCollapseEos
+
+#pragma once
+
+#include <hdf5.h>
+#include <string>
+#include <vector>
+
+#include "DataStructures/BoostMultiArray.hpp" // IWYU pragma: keep
+#include "IO/H5/Object.hpp"
+#include "IO/H5/OpenGroup.hpp"
+
+// IWYU pragma: no_include <boost/multi_array.hpp>
+
+namespace h5 {
+/*!
+ * \ingroup HDF5Group
+ * \brief Reads in tabulated equation of state file from
+ * [stellarcollapse.org](https://stellarcollapse.org)
+ *
+ * Reads in H5 file containing data for tabulated equation of state.
+ * Contains functions to obtain thermodynamic quantities from the file,
+ * stored as either rank-1 or rank-3 datasets.
+ *
+ * It is assumed that the file is in the format of the
+ * [SRO (Schneider, Roberts, Ott 2017) Equation of State files](https://
+ * stellarcollapse.org/SROEOS)
+ *
+ * The description of each dataset in the file can be found
+ * [here](https://bitbucket.org/andschn/sroeos/src/master/
+ * User_Guide/User_Guide.pdf?fileviewer=file-view-default)
+ *
+ */
+class StellarCollapseEos : public h5::Object {
+ public:
+  /// \cond
+  // The stellar collapse files do not have an extension when they are at the
+  // root level ('/') and we currently do not have any combined stellar
```

Comment:
I don't understand what this is trying to say... what do you mean by "file" ?

Reply:
One of the files from https://stellarcollapse.org/SROEOS

Reply:
I don't think we should ever put these files in "/". Do you mean to say that the root-level HDF5 group in the file does not have an extension in its group name?

Reply:
Maybe it would be more helpful if I said "the SRO Equation of State" files?

Reply:
Sorry I made the last comment before your second comment loaded. Yes, that is what I mean.